### DISCUSSION TITLE
Popovers can't be dismissed after opening issue

### DISCUSSION THREAD

**Discussion by nilehmann** [community/community#44367] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

After opening an issue popovers can't be dismissed. This is in firefox.

As you can see in the video. Popovers are first working properly, but after opening the issue popovers are only dismissed when a new one is opened and otherwise will remain always visible. This makes GitHub almost unusable.

[Screencast from 01-14-2023 06:56:44 PM.webm](https://user-images.githubusercontent.com/2400776/212520638-b6e73354-e1b6-4ab2-9222-81587977e12f.webm)

**Comment by foss-** [comment]
Seeing the same all over the place since 2 or 3 days. macOS 13.1 Firefox 108.0.2. Think this was introduced by a change on GitHub not Firefox.

https://user-images.githubusercontent.com/5708172/212570805-f7c1b917-f3d4-490b-ac70-c91d2357dbcd.mov

**Comment by floscher** [comment]
I also see this for a few days now in Firefox. It can be extremely annoying, since the popover gets in the way and doesn't close unless you reload the site. Sometimes it even opens without me hovering over anything.

Firefox 108.0.2 on Ubuntu.

**Comment by juliojgd** [comment]
Same here, this popups hide (e.g.) the merge button and even if I scroll up or down, the popup is there open, so I cannot use the merge button without reloading the page. It's very disturbing when doing  my work!

**Comment by tomcw** [comment]
It appears to be fixed in FF v109.0

**Reply by floscher** [reply]
I was able to still reproduce the issue with Firefox v109.0.

It doesn't happen always, but only sometimes. I can still relatively reliably provoke it by hovering different usernames on PR pages or discussions (like this one, where I'm writing this comment).

But I'm no longer able to get it on the list of pull requests for a repository, where I'm pretty sure that it happened there in the past.

**Reply by floscher** [reply]
~~Update: Wait a second, I just did a hard reload of GitHub.com (Ctrl+Shift+R) and the issue seems to be gone. I even went back and reverted to Firefox 108 and the issue is still gone.~~

~~So it seems GitHub is rolling out a fix at the moment. :tada:~~

Edit: Nevermind, it just happened again on a pull request page (on FF 109).

**Comment by floscher** [comment]
According to discussion #44630 about the same topic, GitHub is working on a fix and they'll notify there when a fix is available:
https://github.com/orgs/community/discussions/44630#discussioncomment-4729878
